### PR TITLE
JSHint

### PR DESCRIPTION
**Pull request by lucaskanashiro**
- Add suport for JSHint checker
- Update travis to run JSHint tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "debile",
+  "version": "0.0.1",
+  "devDependencies": {
+    "jshint": "2.6.x",
```
Comment (sylvestre) [inline package.json]: Sorry, remove the trailing "," here

**Comment by sylvestre** [comment]
Good, the jshint install has been fixed. The rest should be easy

**Comment by lucaskanashiro** [comment]
@sylvestre I ran the tests locally with the same version of JSHint installed in travis and everything is OK, i do not know why these tests are broken in travis :/

The output of the return is coming None, and locally not. Is it a OSError exception handled in run_command?

**Comment by sylvestre** [comment]
Anyway, the application should not crash if the results is none. You
should instead catch the error and shows a clear error message

Thanks
S

**Comment by lucaskanashiro** [comment]
I can handled it, but in this test case it should not be None, I ran in different environments and it worked. And the another checkers does not handle if the output is None,  but I will do it. Anyway,  I will try to check why it happens. 

Thanks.

**Comment by clemux** [comment]
Lucas Kanashiro writes:

> I can handled it, but in this test case it should not be None, I ran in different environments and it worked. And the another checkers does not handle if the output is None,  but I will do it. Anyway,  I will try to check why it happens. 

You can start by logging stderr when the return code is not 0.

**Pull request merged**